### PR TITLE
full course notes added 🎉 by Eva Dee

### PR DESCRIPTION
**Pull request by edieblu**
Adding full course notes. Let me know what you think 😃

### CODE REVIEW

File: building-maps-with-react-leaflet/Lesson07/41-adding-restaurant-information-to-our-popups.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+              <li>
+                <strong>Phone:</strong> ${phone}
+              </li>
+              <li>
+                <strong>Website:</strong> <a href="${website}">${website}</a>
+              </li>
+            </ul>
+          </div>
+        `;
+```
```
Comment (lsminter) [inline building-maps-with-react-leaflet/Lesson07/41-adding-restaurant-information-to-our-popups.md:35]: The code box here needs to be tabbed back a few spaces.

File: building-maps-with-react-leaflet/Lesson09/54-add-the-default-shadows-back-to-our-markers.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`import markerShadow from 'leaflet/dist/images/marker-shadow.png';`
+
+Then add the two shadow configurations to the icon options object:
+
+```js
+      shadowUrl: markerShadow,
+      // if you're using a different icon, you'll probably have to play around with the values a bit to get it right
+      shadowAnchor: [13, 28],
+```
```
Comment (lsminter) [inline building-maps-with-react-leaflet/Lesson09/54-add-the-default-shadows-back-to-our-markers.md:17]: Tab this back a few spaces as well.

File: building-maps-with-react-leaflet/Lesson07/41-adding-restaurant-information-to-our-popups.md (hunk 1 above)
Comment (edieblu) [reply building-maps-with-react-leaflet/Lesson07/41-adding-restaurant-information-to-our-popups.md:35]: Will do, thanks 👍

**Comment by zacjones93** [comment]
Hey Eva! I like the use of the folders to section the notes. Could you make the folder names more semantic? Was thinking something along these lines: 
```
 01 - A Little Overview About Maps
 02 - Add React Leaflet Map to a React App
 03 - Customize with Mapbox Basemap Style
 04 - Adding a Marker to a Map to Point to a Location
 05 - Leaflet State in a React App with Hooks
 06 - Your First GeoJSON Document
 07 - GeoJSON Documents and Tooltips
 08 - Shaded Regions
 09 - Location Markers with Custom Images
 10 - Leaflet's Geolocation API
```

**Comment by edieblu** [comment]
Thanks for the review @lsminter, just pushed the changes 👍

**Comment by edieblu** [comment]
Heya @zacjones93 for sure! I wasn't sure if that breakdown would work so I forgot to put any thought into naming the lesson folders. Will fix 👍

**Comment by edieblu** [comment]
@zacjones93 thanks for the suggestion, directories have been renamed ✅

**Comment by lsminter** [comment]
Last 2 things! We need the notes in a `notes` folder and the renaming/restructuring of the files breaks the table of contents in the README 😅

**Comment by edieblu** [comment]
> Last 2 things! We need the notes in a `notes` folder and the renaming/restructuring of the files breaks the table of contents in the README 😅

Should be good now, thanks! 👍

**Pull request merged**